src/explorer/WebAppProvider.ts

```diff
@@ -31,22 +31,18 @@ export class WebAppProvider implements IChildProvider {
 
         this._nextLink = webAppCollection.nextLink;
 
-        return await Promise.all(webAppCollection
-            .filter((s: Site) => {
+        const treeItems: IAzureTreeItem[] = [];
+        await Promise.all(webAppCollection
+            .map(async (s: Site) => {
                 try {
-                    const siteClient = new SiteClient(s, node);
-                    return siteClient !== undefined;
-                    // tslint:disable-next-line:no-unsafe-any
-                } catch (err) {
-                    return false;
+                    const siteClient: SiteClient = new SiteClient(s, node);
+                    const appServicePlan: AppServicePlan = await siteClient.getAppServicePlan();
```

Comment:
Wrap `getAppServicePlan` and `treeItems.push` in `if (!siteClient.isFunctionApp)`

We have a completely separate extension that handles function apps: https://github.com/Microsoft/vscode-azurefunctions

Reply:
Ugh, fixed... forgot about that.